compiler-core/src/javascript/pattern.rs

```diff
@@ -409,6 +409,7 @@ impl<'module_ctx, 'expression_gen, 'a> Generator<'module_ctx, 'expression_gen, '
         subject: &Document<'a>,
         pattern: &'a TypedPattern,
     ) -> Result<(), Error> {
+        println!("traverse_pattern! {:?}", pattern);
```

Comment:
Debug print here :)
(If you use `dbg!`, clippy will warn you!)

Reply:
oh no, good spot, thank you :purple_heart: 

you see the thing is I also have to disable clippy otherwise rust-analyser would be literally unusable and its lockfile also blocks recompilation/running the tests

Reply:
I'm struggling here ok

Reply:
I just run it once using `cargo clippy` before pushing, after tests pass